src/DiscordServer.ts

```diff
@@ -244,7 +244,7 @@ export class DiscordServer {
         }
 
         if (data.length > 0 && data != null) {
-          var textMatcher = new RegExp(/(?<=\[Text]\n)(.*)(?:=\n\[Options|\n\n)/ms)
+          var textMatcher = new RegExp(/^((?!\[Options]|\[Text]).)*$/ms)
```

Comment:
With this new Regex, we only match between `[Text]` and `[Options]` but doesn't fail if `[Options]` don't exist, and doesn't match `[Options]` if there are multiple new lines below it.